### ISSUE TITLE
IPC category

### ISSUE THREAD

**Issue by Grinnz** [EnlightenedPerlOrganisation/task-kensho#56]
An IPC category would be useful for recommendations on modules to handle running and managing processes. IPC::Run, IPC::Run3, IPC::System::Simple, and maybe even IO::Async::Process or POE::Wheel::Run.

**Comment by ttkciar** [comment]
These modules sound like they are more about managing processes than facilitating interprocess communication.  The pipes used to shuffle data to/from those processes are of course IPC, but they seem like their secondary focus.

Maybe put these modules into a "Multiprocessing" category and fill the "IPC" category with modules which facilitate signals, shared memory, message-passing, etc?

**Comment by Grinnz** [comment]
I agree the IPC name isn't ideal. It just happens to be the namespace used for that on CPAN. Maybe something along the lines of "Command Management". I think it should be distinct from process management which is more like what Parallel::ForkManager and such do.

**Comment by ttkciar** [comment]
You're totally right that CPAN (mis)uses the IPC:: namespace for process-wrangling.  It's probably worth continuing in the same vein to remain consistent.

**Comment by davebaird** [comment]
System::Command and Proc::Background provide very convenient and friendly object wrappers to system commands that probably cover 90% of use cases.
